### PR TITLE
Add docs for Notion

### PR DESCRIPTION
**Pull request by bachya**
**Description:**

This PR adds docs for the forthcoming `notion` component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** https://github.com/home-assistant/home-assistant/pull/24634

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_components/notion.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+title: "Notion"
+description: "How to integrate Notion kits within Home Assistant."
+date: 2019-06-19 20:52
+sidebar: true
+comments: false
+logo: notion.png
+ha_category:
+  - Hub
+  - Sensor
+ha_release: 0.95
```
Comment (MartinHjelmare) [inline source/_components/notion.markdown]: 0.96

File: source/_components/notion.markdown [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+layout: page
+title: "Notion"
+description: "How to integrate Notion kits within Home Assistant."
+date: 2019-06-19 20:52
+sidebar: true
+comments: false
+logo: notion.png
+ha_category:
+  - Hub
+  - Sensor
```
Comment (MartinHjelmare) [inline source/_components/notion.markdown:12]: There's a binary sensor platform too right?

File: source/_components/notion.markdown [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ha_release: 0.95
+ha_iot_class: Cloud Polling
+---
+
+The `Notion` component retrieves data from [Notion](https://getnotion.com) wireless
+home monitoring sensor kits.
+
+## {% linkable_title Configuration %}
```
Comment (klaasnicolaas) [inline source/_components/notion.markdown]:
```suggestion
## Configuration
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by MartinHjelmare — APPROVED** [review]

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
Time to merge it. Thnx for the PR! 👍

**Pull request merged**